src/main/java/module-info.java

```diff
@@ -31,12 +32,13 @@
 	requires com.tobiasdiez.easybind;
 	requires dagger;
 	requires io.github.coffeelibs.tinyoauth2client;
+	requires libappindicator.gtk3.minimal.java;
```

Comment:
I would strongly suggest to align the module name with the package name and make sure that both follow reverse domain name notation for a domain that you control. Otherwise we might have to deal with split packages again.